pkg/resource/alias/hooks.go

```diff
@@ -183,6 +204,8 @@ func (rm *resourceManager) getFunctionEventInvokeConfig(
 	return nil
 }
 
+// setResourceAdditionalFields will describe the fields that are not return by
```

Comment:
I think the comment would be more accurate if it said ".. the fields that are not returned by **the getFunctionConfiguration** API call" (Considering that the info here is still returned by API calls)

Reply:
Thanks, that makes more sense.